Week3/homework/step2-3.js

```diff
@@ -31,9 +35,11 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  let i = 1;
+  do {
```

Comment:
What happens if `num <= 0`? The loop body will still be executed once. This is because in a `do-while` loop the loop condition is tested at the end of the loop, making the `do-while` loop rather useless. You can always rewrite a `do-while` loop as a regular `while` loop. Using a `do-while` loop is like having unsafe sex on your first date 😁.

Since we are asking you here to use a `do-while` loop all the same, you need to add an `if` statement to check the value of `num` and determine whether the loop should be executed at all.